package.json

```diff
@@ -47,17 +46,13 @@
     "babel-core": "6.26.3",
     "babel-loader": "6.4.1",
     "babel-preset-es2015": "6.24.1",
-    "bench": "0.3.6",
-    "benchmark": "2.1.4",
     "browserify": "14.5.0",
     "browzers": "1.3.0",
-    "bulk-require": "1.0.1",
```

Comment:
I thought this was used for the other tests?

Reply:
Yes maybe, which one? I didn't found any occurrences of the import anymore. I've replaced the runner implementation on https://github.com/algolia/algoliasearch-helper-js/pull/666.

Reply:
was confused about the order of the PRs